tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -514,6 +516,8 @@ handle_build() {
 
 	tdvf) install_tdvf ;;
 
+	ovmf) install_ovmf ;;
```

Comment:
@fitzthum, we're trying to keep those alphabetically sorted. Mind to move this entry after nydus-tarball and before qemu-tarball ?